src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,25 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class StringTest {
+    @Test
+    void split(){
+        String[] result = "1,2".split(",");
+        assertThat(result).containsExactly("1","2");
+    }
+    @Test
+    void substring(){
+        String result = "(1,2)".substring(1,4);
+        assertThat(result).isEqualTo("1,2");
+    }
+
+    @Test
+    void charAt() throws Exception{
+        assertThrows(StringIndexOutOfBoundsException.class,()->{char result = "abc".charAt(4);} );
+    }
```

Comment:
```suggestion
    void charAt() {
        assertThatThrownBy(() -> {
            "abc".charAt(3);
        }).isInstanceOf(StringIndexOutOfBoundsException.class);
    }
```

학습 테스트의 힌트를 사용하면 이런 방식으로도 예외를 확인해볼 수 있어요. :)